### PR TITLE
[MB-15174] render weight warning on review shipment weights page

### PR DESCRIPTION
**Pull request by ruizajtruss**
## Summary

This PR adds a warning alert to the `review shipment weights` page; which should only render when a move has excess weight

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Login as a service counselor
2. Access this move `http://officelocal:3000/counseling/moves/XSWT01/review-shipment-weights` or any move that's labeled to have excess weight
3. The alert should be present at the top of the page

## Verification Steps for Author

These are to be checked by the author.

- [x] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/main/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

## Screenshots
image

### CODE REVIEW

**Review by AnnaGingle — DISMISSED** [review]
Looked at multiple moves with excess weight No banner on review shipment weights page.

**Comment by ruizajtruss** [comment]
> Looked at multiple moves with excess weight No banner on review shipment weights page.

I thought maybe merging other changes might have broken things but everything works as expected locally. I think it may have just been a funky deploy. I took the liberty of deploying it again and it appears to be working 

https://user-images.githubusercontent.com/111928238/225811424-598de3c0-8c5b-4f9f-ae9a-b984c91c5399.mp4

File: src/pages/Office/ServicesCounselingReviewShipmentWeights/ServicesCounselingReviewShipmentWeights.jsx [hunk 1]
```diff
@@ -39,7 +39,9 @@ const sortShipments = (shipments) => {
   });
   return { hhgShipment, ppmShipment };
 };
+
 const ServicesCounselingReviewShipmentWeights = ({ moveCode }) => {
+  const [showExcessWeightAlert, setShowExcessWeightAlert] = useState(null);
```
Comment (michaelvaldes-truss) [inline src/pages/Office/ServicesCounselingReviewShipmentWeights/ServicesCounselingReviewShipmentWeights.jsx:44]: **nitpick (non-blocking):** you can set the initial state to "false" to clarify that this is boolean

File: src/pages/Office/ServicesCounselingReviewShipmentWeights/ServicesCounselingReviewShipmentWeights.test.jsx [hunk 2]
```diff
@@ -94,5 +97,24 @@ describe('Services Counseling Review Shipment Weights', () => {
...
+      await render(<ServicesCounselingReviewShipmentWeights moveCode="XSWT01" />);
+
+      const excessWeightWarning = await screen.findByTestId('alert');
+      expect(excessWeightWarning).toBeInTheDocument();
+      expect(excessWeightWarning).toHaveTextContent(
+        'This move has excess weight. Review PPM weight ticket documents to resolve.',
+      );
+    });
+
+    it('does NOT display excess weight warning when move has excess weight', async () => {
```
Comment (michaelvaldes-truss) [inline src/pages/Office/ServicesCounselingReviewShipmentWeights/ServicesCounselingReviewShipmentWeights.test.jsx:112]: **suggestion:** the text here should say "...when move does NOT have excess weight"

**Review by michaelvaldes-truss — APPROVED** [review]
Looks good! Just added a couple of small suggestions.

**Pull request merged**